### PR TITLE
Update zh translation in basic-stateful-set.md

### PR DESCRIPTION
**Pull request by wwgfhf**
Update zh translation in basic-stateful-set.md

### CODE REVIEW

File: content/zh/docs/tutorials/stateful-application/basic-stateful-set.md [hunk 1]
```diff
@@ -356,7 +356,7 @@ to Running and Ready.
 Pod 的序号、主机名、SRV 条目和记录名称没有改变，但和 Pod 相关联的 IP 地址可能发生了改变。在本教程中使用的集群中它们就改变了。这就是为什么不要在其他应用中使用 StatefulSet 中的 Pod 的 IP 地址进行连接，这点很重要。
 
 
-如果你需要查找并连接一个 StatefulSet 的活动成员，你应该查询 Headless Service 的 CNAME。和 CNAME 相关联的 SRV 记录只会包含 StatefulSet 中处于 Running 和 Ready 状态的 Pod。
+如果你需要查找并连接一个 StatefulSet 的活动成员，你应该查询 Headless Service (`nginx.default.svc.cluster.local`) 的 CNAME。和 CNAME 相关联的 SRV 记录只会包含 StatefulSet 中处于 Running 和 Ready 状态的 Pod。
```
Comment (tengqm) [inline content/zh/docs/tutorials/stateful-application/basic-stateful-set.md:359]:
```suggestion
如果你需要查找并连接一个 StatefulSet 的活动成员，你应该查询 Headless Service 的 CNAME （`nginx.default.svc.cluster.local`）。和 CNAME 相关联的 SRV 记录只会包含 StatefulSet 中处于 Running 和 Ready 状态的 Pod。
```

Comment (tanjunchen) [reply content/zh/docs/tutorials/stateful-application/basic-stateful-set.md:359]: @wwgfhf 
any progress?

Comment (tanjunchen) [reply content/zh/docs/tutorials/stateful-application/basic-stateful-set.md:359]: /cc @wwgfhf

**Comment by tengqm** [comment]
This has been inactive for about 2 months. Pls feel free to reopen.
/close

**Pull request closed without merging**